testutil/network/sessionnet/network.go

```diff
@@ -0,0 +1,85 @@
+package sessionnet
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/testutil/network"
+	"github.com/pokt-network/poktroll/testutil/network/basenet"
+	"github.com/pokt-network/poktroll/testutil/testkeyring"
+)
+
+var _ network.InMemoryNetwork = (*inMemoryNetworkWithSessions)(nil)
+
+// inMemoryNetworkWithSessions is an implementation of the InMemoryNetwork interface.
+type inMemoryNetworkWithSessions struct {
+	basenet.BaseInMemoryNetwork
+}
+
+// DefaultInMemoryNetworkConfig returns the default in-memory network configuration.
+// This configuration should sufficient populate on-chain objects to support reasonable
+// coverage around most session-oriented scenarios.
+func DefaultInMemoryNetworkConfig(t *testing.T) *network.InMemoryNetworkConfig {
+	t.Helper()
+
+	return &network.InMemoryNetworkConfig{
+		NumSessions:             4,
+		NumRelaysPerSession:     5,
+		NumSuppliers:            2,
+		AppSupplierPairingRatio: 2,
```

Comment:
Same as above - can we have a comment explaining this field and how it works.

Reply:
https://github.com/pokt-network/poktroll/blob/main/testutil/network/types.go#L46